release-notes/release-notes59.md

```diff
@@ -85,10 +85,10 @@ For information on these commands, see: [Table - Commands]({{site.baseurl}}/plug
 : Toggles the caption on the selected table.
 
 `tablecellbackgroundcolor`
-: Allows the user to change the background color of selected cells. The available colors are set by the [`table_cell_background_color_map` option]({{site.baseurl}}/plugins/opensource/table/#table_cell_background_color_map) or [`color_map` option]({{site.baseurl}}/plugins/opensource/table/#color_map).
+: Allows the user to change the background color of selected cells. The available colors are set by the [`table_background_color_map` option]({{site.baseurl}}/plugins/opensource/table/#table_background_color_map) or [`color_map` option]({{site.baseurl}}/plugins/opensource/table/#color_map).
```

Comment:
Will we mention anywhere that 'table_cell_background_color_map' was superceeded by the new 'table_background_color_map' setting?

Reply:
No, as that was never released.